scss/_progress.scss

```diff
@@ -1,32 +1,37 @@
+// Progress animations
```

Comment:
You can drop these newly added comments—they don't add any new context that the code itself doesn't already tell you :).